corehq/apps/api/resources/v0_5.py

```diff
@@ -857,9 +858,9 @@ def obj_get_list(self, bundle, **kwargs):
             results.append(Form(form_xmlns=form.xmlns, form_name=form_name))
         return results
 
-#Zapier requires id and name; case_type has no obvious id, placeholder inserted instead.s
-Case = namedtuple('Case', 'case_type placeholder')
-Case.__new__.__defaults__ = ('', '')
+# Zapier requires id and name; case_type has no obvious id, placeholder inserted instead.
+CaseType = namedtuple('Case_Type', 'case_type placeholder')
```

Comment:
I think the convention is for a `namedtuple` to have the same typename as the name of the variable you are assigning it to, so this should be `CaseType`, not `Case_Type` (and as a general rule, you probably won't see `Things_Like_This` in python